kNN_NBC/NBC/nbc.h

```diff
@@ -9,3 +9,5 @@
 
 int DLL_EXPORT fun(int x);
 std::vector<int> DLL_EXPORT fun_negate(std::vector<int> arg);
+
+std::vector<int> DLL_EXPORT nbc_kNN(std::vector<std::vector<double>> attributes);
```

Comment:
można spróbować przekazywać przez referencje, swig powinien to łyknąć

Reply:
swig nie bardzo chce współpracować z referencjami albo nie wiem jak mu to podać

Reply:
a const reference? chyba nie będziemy zmieniać punktów

Reply:
spróbuję

Reply:
chyba działa